### PR TITLE
Update DG with Undo and Redo Use Cases

### PR DESCRIPTION
**Pull request by bchenghi**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -737,6 +737,43 @@ Use case ends.
...
+#### UC11: Undo most recent undoable command
+
+**Guarantees** The most recent undoable command (add, delete, edit, sort, clear) requested will be undone.
+
+**MSS:**
+
+**1.** User requests to undo.
+
+**2.** SimplyKitchen displays the food list before the most recent undoable command was requested.
+
```
Comment (songyiang) [inline docs/DeveloperGuide.md:749]: Should there be a "Use case ends." at the end of MSS? Same for line 767.

**Review by songyiang — CHANGES_REQUESTED** [review]
Just a minor comment.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (bchenghi) [reply docs/DeveloperGuide.md:749]: Thanks! Updated.

**Review by songyiang — APPROVED** [review]
LGTM

**Pull request merged**